driver/include/esp_camera.h

```diff
@@ -112,8 +112,14 @@ typedef struct {
     int pin_pwdn;                   /*!< GPIO pin for camera power down line */
     int pin_reset;                  /*!< GPIO pin for camera reset line */
     int pin_xclk;                   /*!< GPIO pin for camera XCLK line */
-    int pin_sscb_sda;               /*!< GPIO pin for camera SDA line */
-    int pin_sscb_scl;               /*!< GPIO pin for camera SCL line */
+    union {
+        int pin_sccb_sda;           /*!< GPIO pin for camera SDA line */
+        int pin_sscb_sda;           /*!< GPIO pin for camera SDA line (legacy name) */
```

Comment:
Thanks for fixing this typo!

I suggest adding `__attribute__((deprecated("please use pin_sccb_sda instead")))` here so that users can notice this change.

Reply:
Done.